.github/workflows/tests.yml

```diff
@@ -132,7 +136,11 @@ jobs:
        with:
          files: report/jest/clover.xml,report/phpunit/junit.jest.xml
          fail_ci_if_error: true
-     - name: Slack Notification
-       uses: rtCamp/action-slack-notify@v2
+     - name: Send Slack notification
+       #if: failure()
+       uses: 8398a7/action-slack@v2
```

Comment:
@gggrant Any reason to use v2 instead of [v3 (latest)](https://github.com/8398a7/action-slack) for the Slack notifications?